### PR TITLE
Add support for nested cloudformation templates

### PR DESCRIPTION
**Pull request by scisco**

### CODE REVIEW

File: packages/deployment/app/config.yml [hunk 1]
```diff
@@ -8,6 +8,14 @@ default:
   message_adapter_repo: cumulus-message-adapter
   message_adapter_filename: cumulus-message-adapter.zip
 
+  nested_templates:
+    CumulusApiDefault:
+      cfFile: node_modules/@cumulus/deployment/app/cumulus_api.template.yml
+      configFile: node_modules/@cumulus/deployment/app/cumulus_api_default.config.yml
+    CumulusApiV1:
+      cfFile: node_modules/@cumulus/deployment/app/cumulus_api.template.yml
```
Comment (abarciauskas-bgse) [inline packages/deployment/app/config.yml]: Should this file be the same as `nested_templates.CumulusApiDefault.cfFile`?

Comment (scisco) [reply packages/deployment/app/config.yml]: yeah, because they share the same template.

Comment (scisco) [reply packages/deployment/app/config.yml]: Basically default and version 1.0 are identical except that they have different api web addresses. V1 has `/v1` in front of all addresses.

File: packages/api/config/api_default.yml [hunk 2]
```diff
@@ -0,0 +1,383 @@
+ApiCollectionsDefault:
```
Comment (abarciauskas-bgse) [inline packages/api/config/api_default.yml]: For my own edification what is the difference between `api_default` and `api_v1`?

File: packages/deployment/package.json [hunk 3]
```diff
@@ -26,7 +26,7 @@
   "author": "Cumulus Authors",
   "license": "Apache-2.0",
   "dependencies": {
-    "kes": "~2.0.3",
+    "kes": "~2.1.0",
```
Comment (abarciauskas-bgse) [inline packages/deployment/package.json]: Probably we also need to update all package versions, given the updates to @cumulus/deployment - but complete understand if you're waiting for reviews to do that for readability

**Pull request merged**